Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmon.yaml

```diff
@@ -113,17 +117,23 @@ ParserQuery: |
                 EventVendor = 'Microsoft',
                 EventSchemaVersion = '0.2.4',
                 EventSchema = 'NetworkSession', 
-                EventProduct = 'Sysmon for Windows',
+                EventProduct = 'Sysmon',
                 EventResult = 'Success',
                 EventSeverity = 'Informational',
                 DvcOs = 'Windows',
                 Protocol = toupper(Protocol),
                 EventOriginalType = '3', // Set with a constant value to avoid parsing
-                SrcDomain = iff(tostring(split(Dvc, "\\")[0]) == Dvc, Dvc, tostring(split(Dvc, "\\")[0])),
+                SrcDomain = iff(tostring(split(SrcHostname, "\\")[0]) == SrcHostname, "", tostring(split(SrcHostname, "\\")[0])),
```

Comment:
1. If you need a value twice in a split, put it in a temp.
2. The definition of Hostname, Domain, DomainType and FQDN is specific and needs to be handled correctly.